app/src/verifier.h

```diff
@@ -70,6 +70,10 @@ namespace scitt::verifier
       auto issuer = phdr.issuer;
       auto kid = phdr.kid;
       auto x5chain = phdr.x5chain;
+      if(issuer.has_value() && !configuration.policy.is_accepted_issuers(issuer.value()))
```

Comment:
I would move this further down into the DID-specific code path, after we've checked that issuer has a value.

@letmaik what should we do if there is both an `x5chain` and an `issuer`? Probably we'll want to flip this logic, where we use the presence of the `issuer` field instead of `x5chain` to decide which code path we follow (since that will allow us to use `did:x509`)

Reply:
Yes, it should be flipped, let's do it in a separate PR, maybe when adding `did:x509`.